molencoder/utils.py

```diff
@@ -21,16 +73,17 @@ def __init__(self, rep):
         self.rep = rep
 
     def forward(self, x):
-        size = (1,) + tuple(x.size())
+        size = tuple(x.size())
+        size = (size[0], 1) + size[1:]
```

Comment:
This seems to be the reason why the output was crap before. Losses are finally decreasing steadily now 😄